src/chess/piece/King.java

```diff
@@ -19,6 +19,17 @@ public boolean isValidMove(Position newPosition) {
       return false;
     }
     // ToDo: implement
```

Comment:
When you finish implementing, you need to remove this `ToDo` comment
This is not just a comment, but kind of a mark